sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2020 Intel Corporation
+
+/*
+ *  sof_sdw_max98373 - Helpers to handle 2x MAX98373 codec devices
+ *  from generic machine driver
+ */
+
+#include <linux/device.h>
+#include <linux/errno.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+#include "sof_sdw_common.h"
+
+static const struct snd_soc_dapm_widget mx8373_widgets[] = {
+	SND_SOC_DAPM_SPK("Left Spk", NULL),
+	SND_SOC_DAPM_SPK("Right Spk", NULL),
+};
+
+static const struct snd_soc_dapm_route mx8373_map[] = {
+	/* Speakers */
+	{ "Left Spk", NULL, "mx8373-1 BE_OUT" },
+	{ "Right Spk", NULL, "mx8373-2 BE_OUT" },
+};
+
+static const struct snd_kcontrol_new mx8373_controls[] = {
+	SOC_DAPM_PIN_SWITCH("Left Spk"),
+	SOC_DAPM_PIN_SWITCH("Right Spk"),
+};
+
+static int spk_init(struct snd_soc_pcm_runtime *rtd)
+{
+	struct snd_soc_card *card = rtd->card;
+	int ret;
+
+	card->components = devm_kasprintf(card->dev, GFP_KERNEL,
+					  "%s spk:mx8373",
```

Comment:
we should add something about the number of amps and number of speakers to help with UCM management, as done for rt1308.

Reply:
shall work with @RanderWang and understand. Thanks

Reply:
@naveen-manohar  amp number is added in sof_sdw.c, but we need to add quirk for volteer to indicate spk number, please check https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/intel/boards/sof_sdw.c#L65.  default spk num = 2

Reply:
Done, verified the change.